testing/metaopt/test_automl.py

```diff
@@ -262,3 +264,30 @@ def test_automl_should_shallow_copy_data_before_each_epoch():
 
     assert isinstance(best_model, Pipeline)
 
+
+def test_trainer_train():
+    data_inputs = np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+    expected_outputs = data_inputs * 4
+    p = Pipeline([
+        MultiplyByN(2).set_hyperparams_space(HyperparameterSpace({
+            'multiply_by': FixedHyperparameter(2)
+        })),
+        NumpyReshape(new_shape=(-1, 1)),
+        linear_model.LinearRegression()
+    ])
+
+    trainer: Trainer = Trainer(
+        epochs=10,
+        scoring_callback=ScoringCallback(mean_squared_error, higher_score_is_better=False),
+        validation_splitter=ValidationSplitter(test_size=0.20)
+    )
+
+    repo_trial: Trial = trainer.train(pipeline=p, data_inputs=data_inputs, expected_outputs=expected_outputs)
+
+    trained_pipeline = repo_trial.get_trained_pipeline(split_number=0)
+
+    outputs = trained_pipeline.transform(data_inputs)
+    mse = mean_squared_error(expected_outputs, outputs)
+
+    assert mse < 1000
```

Comment:
really, 1000 ? This test doesn't seem overly random, it looks to me that a smaller value could be used.